packages/collections/test.pony

```diff
@@ -431,7 +432,6 @@ class iso _TestListsContains is UnitTest
     h.assert_eq[Bool](a.contains(3), false)
 
     true
-
```

Comment:
Just a little formatting thing - I had added a trailing newline where the previous version of the file didn't have one.